bot_modules/tags/loader.js

```diff
@@ -0,0 +1,69 @@
+const path = require('path')
+const fs = require('fs')
+
+// Relative to the project root
+const tagFolder = './tags'
+
+/**
+ * Load the tags from file and return them
+ */
+exports.load = () => {
+  const tags = {}
+
+  // Read tags from files
+  fs.readdirSync(tagFolder).forEach(file => {
+    if (!file.endsWith('.tag')) {
```

Comment:
Weren't we considering making it wiki-based?

Reply:
We opted for this file method as it allows much more complex tags and PRs to add more.